src/NServiceBus.Transport.Msmq.Tests/InstanceMapping/InstanceMappingFileFeatureTests.cs

```diff
@@ -11,13 +11,15 @@ public class InstanceMappingFileFeatureTests
         [Test]
         public void Should_configure_default_values()
         {
-            var feature = new InstanceMappingFileFeature();
-            var settings = new SettingsHolder();
+            // Test has to be commented out for now, no way to access the defaults
```

Comment:
I see what you mean now. It seems reasonable to me to expose the registered defaults in the core?

I can handle this as a separate PR if needed? cc @Particular/nservicebus-maintainers